packages/apps/desktop-viewer-test/src/frontend/components/home/Recents.tsx

```diff
@@ -20,8 +20,14 @@ export const Recents = () => {
     }
   }, [userSettings]);
 
-  const openFile = async (filePath?: string) => {
-    await navigate(`/viewer`, { state: { filePath } });
+  const openFile = async (file?: ViewerFile) => {
```

Comment:
i was wondering the other day why file is optionally defined. idk if that makes sense